packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -106,6 +106,15 @@ const MediaControls = kind({
 		 */
 		moreButtonCloseLabel: PropTypes.string,
 
+		/**
```

Comment:
Why don't we link or cross-reference this to `IconButton`'s `color` property?  And, perhaps indicate that it's not just the underline but that it links to the color button the remote.